app/views/candidate_interface/equality_and_diversity/edit_disabilities.html.erb

```diff
@@ -6,7 +6,7 @@
     <%= form_with model: @disabilities, url: candidate_interface_update_equality_and_diversity_disabilities_path, method: :post do |f| %>
       <%= f.govuk_error_summary %>
 
-      <%= f.govuk_check_boxes_fieldset :disabilities, legend: { size: 'xl', text: equality_and_diversity_caption + 'Please select all that apply to you' } do %>
+      <%= f.govuk_check_boxes_fieldset :disabilities, multiple: false, legend: { size: 'xl', text: equality_and_diversity_caption + 'Please select all that apply to you' } do %>
```

Comment:
Just a note on this and beyond the scope of this PR, but `govuk_check_boxes_fieldset` [now takes a `caption` argument](https://govuk-form-builder.netlify.app/introduction/labels-captions-hints-and-legends/#a-label-with-a-caption), which will make this a little easier on the eye.